Server/Game/tasks.py

```diff
@@ -4,122 +4,123 @@
+            logger.info(str(first_player.user.username) + " will take the first turn")
+            player_turn = first_player.user
+        elif player_turn_index == 2:
+            logger.info(str(second_player.user.username) + " will take the first turn")
+            player_turn = second_player.user
 
-    if not mock_maps:
         maps = Map.objects
-    else:
-        maps = mock_maps
 
-    # Randomly choose a map for the game
-    index = random.randint(0, maps.count() - 1)
-    game_map = maps.filter()[index]
+        # Randomly choose a map for the game
+        index = random.randint(0, maps.count() - 1)
+        game_map = maps.filter()[index]
 
-    logger.info("Map chosen for this game: " + str(game_map))
+        logger.info("Map chosen for this game: " + str(game_map))
 
-    # Create the game
-    game = Game(user_turn=player_turn, version=version, map_path=game_map)
-    game.save()
+        # Create the game
+        game = Game(user_turn=player_turn, version=version, map_path=game_map)
+        game.save()
 
-    logger.info("Created the game: " + str(game))
+        logger.info("Created game id: " + str(game.id))
 
-    if not mock_game_users:
         game_users = Game_User.objects
-    else:
-        game_users = mock_game_users
 
-    logger.debug("Updating game_user entries")
+        logger.debug("Updating game_user entries")
 
-    # Link the game to the first player's game entry
-    game_user1 = game_users.filter(user=first_player.user, game=None).first()
-    game_user1.game = game
-    game_user1.name = "vs. " + str(second_player.user.username) + " #" + str(count + 1)
-    game_user1.save()
+        # Link the game to the first player's game entry
+        game_user1 = game_users.filter(user=first_player.user, game=None).first()
+        game_user1.game = game
+        game_user1.name = "vs. " + str(second_player.user.username) + " #" + str(count + 1)
+        game_user1.team = 1
+        game_user1.save()
 
-    # Link the game to the second player's game entry
-    game_user2 = game_users.filter(user=second_player.user, game=None).first()
-    game_user2.game = game
-    game_user2.name = "vs. " + str(first_player.user.username) + " #" + str(count + 1)
-    game_user2.save()
+        # Link the game to the second player's game entry
+        game_user2 = game_users.filter(user=second_player.user, game=None).first()
+        game_user2.game = game
+        game_user2.name = "vs. " + str(first_player.user.username) + " #" + str(count + 1)
+        game_user2.team = 2
+        game_user2.save()
 
-    logging.debug("Updating each player's unit entries")
+        logging.debug("Updating each player's unit entries")
 
-    # Update each player's units
-    for unit in Unit.objects.filter(owner=first_player.user, game=None):
-        unit.game = game
-        unit.save()
+        # Update each player's units
+        for unit in Unit.objects.filter(owner=first_player.user, game=None):
```

Comment:
In the name of cleanup, could this and the below for loops be combined with something like:
filter(owner__in=[owner1, owner2]) ?